styles/pbwow3/template/event/index_body_forumlist_body_after.html

```diff
@@ -0,0 +1,3 @@
+{% if MCHAT_INDEX and not MCHAT_LOCATION %}
+	{% INCLUDE '@dmzx_mchat/mchat_body.html' %}
+{% endif %}
```

Comment:
This file is the same as prosilver's, it shouldn't be necessary to have it here too.